lib/sorted/sql_query.rb

```diff
@@ -5,7 +5,7 @@ module Sorted
   class SQLQuery
     extend Parse
 
-    REGEXP = /(([a-z0-9._]+)\s([asc|desc]+)|[a-z0-9._]+)/i
+    REGEXP = /(([a-z0-9._]+)\s([asc|desc]+)\s?(nulls\s[first|last]+)?|[a-z0-9._]+)/i
```

Comment:
does this cover `order by column nulls first`? (without asc/desc)

Reply:
Just changed it to cover this case 👍